_posts/blog/2019-10-17-2019-10-17-lab-meeting.md

```diff
@@ -0,0 +1,159 @@
+>
+> **There's a role for everyone in a team and these are equally important.**
+> Even if it's reviewing code or writing documentation, these are as important (arguably, more so) than the code itself.
+> Code that does what you think it's doing and has been explained well will have a much longer lifespan than code that is difficult to follow, regardless of how clever it is, if poorly documented.
+>
+> **I found my strength as a leader.**
+> You might remember at the beginning of this blog post I mentioned that I'd never led a hack project before, so I also learned a lot about my leadership capabilities.
+> I think I did well.
+> I identified the strengths of each of my team members and gave them a task suited to them whilst letting them explore new concepts, offering my own insight and opinions when required.
+>
+> **Hackathons are not where projects end.**
+> While the first 80% of a project to get the infrastructure in place can be achieved in a short amount of time, like a hackathon, the last 20% is hard and often takes longer.
+> But this extra effort is necessary for the software to be taken up by others.
+
+**Patricia** was frustrated with the way a recent hackathon was facilitated.
+She was reflecting on how sensitive she is to being made to feel welcome and how difficult that can be at larger conferences.
+The main problem is when folks are using the event both for collaborative working _and_ for hanging out with their friends.
+It ends up being really hard to join conversations and meet new people.
+(*Sorry you had a rubbish time - turns out running an unconference or hackathon is harder than most people think! --KW*)
+
+**Yini** recommended a Coursera course on ["Improving Your Statistical Questions"](https://www.coursera.org/learn/improving-statistical-questions)
+
+**Ang** is still working on a revision 😓😓😓 and is looking forward to being more sociable in the near future.
+
+**Kirstie** really enjoyed the Autistica [Artificial Intelligence and Early Intervention Summit](https://www.autistica.org.uk/get-involved/events/ai-summit).
+It was two days bringing together AI researchers and data scientists, autism researchers, clinicians, autistic adults and parents of autistic children.
+The goal was to dream big about how we can use AI to support early intervention in autism, and then brainstorm grant proposals to be submitted next month.
+Turns out people liked her citizen science idea....so stay tuned for a proposal going in soon! 😅
+
+**Elizabeth** is learning more with [Vue.js](https://vuejs.org), and she shared a really wonderful experience that reminded her why licensing is so important!
+Someone had openly published code to do _close_ to what I wanted [https://github.com/harps116/vue-github](https://github.com/harps116/vue-github), but because they licensed it under an MIT license I was able to remix and extend it without any concerns 💃💃
```

Comment:
```suggestion
Someone had openly published code to do _close_ to what she wanted [https://github.com/harps116/vue-github](https://github.com/harps116/vue-github), but because they licensed it under an MIT license she was able to remix and extend it without any concerns 💃💃
```